src/shared/scrapers/US/MO/index.js

```diff
@@ -168,6 +194,59 @@ const scraper = {
 
     return countyName;
   },
+  // Missouri stores test result data separately from death and case counts, which are reported on / updated only
+  // daily as cumulative counts. This function downloads the cumulative testing counts for current scrape
+  // date.
+  async _applyTestingCounts(counties) {
+    const captureDate = new Date(datetime.scrapeDate() || getYYYYMMDD());
+
+    const queryParams = {
+      args: {
+        where: `test_date <= DATE '${getYYYYMMDD(captureDate)}'`,
+        groupByFieldsForStatistics: 'county,result',
```

Comment:
This query relies on arcGIS to do the aggregation, at least to the county and test result (Positive, Negative, Indeterminate) level. Incidentally, the positive case counts in this data set differs slightly from the historically, snapshotted data previously collected by this scraper (these positive case counts are slightly higher). It's possible that the snapshotted counts are lower due to reporting lag, but I'm curious which of the counts would be preferred in the report.